prometheus/Tiltfile

```
@@ -0,0 +1,61 @@
+
+# Download the manifest and apply it.
+bundle = local("curl -sSL https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/v0.41.1/bundle.yaml", quiet=True)
```

```suggestion
bundle = local("curl -fsSL https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/v0.41.1/bundle.yaml", quiet=True)
```

without `-f`, curl will exit with status 0 even on 4xx/5xx

---

yes! added